Resources/Prototypes/Entities/Objects/Power/lights.yml

```diff
@@ -1,4 +1,26 @@
 ﻿- type: entity
+  parent: BaseItem
+  name: baselightbulb
```

Comment:
name isn't necessary in abstracts

Reply:
This was how it was already, will remove though.